checker/tests/fenumswing/IdentityArrayList.java

```diff
@@ -1,10 +1,14 @@
 import java.util.Arrays;
+import org.checkerframework.checker.fenum.qual.FenumTop;
 
 /*
  * This test case violates an assertion in the compiler.
  * It does not depend on the Fenum Checker, it breaks for any checker.
  */
 public class IdentityArrayList {
+    // The type the third argument to Arrays.copyOf should be:
```

Comment:
English typo.
Also, "should be" suggests a limitation/bug somewhere.  Can you be explicit about where that bug/limitation is?